aws/resource_aws_elasticache_replication_group.go

```diff
@@ -310,6 +344,9 @@ func resourceAwsElasticacheReplicationGroupRead(d *schema.ResourceData, meta int
 		}
 
 		d.Set("auto_minor_version_upgrade", c.AutoMinorVersionUpgrade)
+		d.Set("at_rest_encryption", c.AtRestEncryptionEnabled)
+		d.Set("transit_encryption", c.TransitEncryptionEnabled)
+		d.Set("auth_token", c.TransitEncryptionEnabled)
```

Comment:
Typo ☝️ 

Because we cannot get the token back from the API (for obvious reasons) I think we should just make it `nil` if `c.AuthTokenEnabled != nil && !*c.AuthTokenEnabled` otherwise leave it alone during `Read()`.

Reply:
ah, great catch!